### ISSUE TITLE
Won't install on win 10

### ISSUE THREAD

**Issue by danielscnc** [cDima/Aerial#93]
I can not get this to install on win 10, It just runs on screen

**Comment by alistairmcmillan** [comment]
What happens when you right-click and choose "Install" from the menu?

**Comment by zarurc** [comment]
Installed this yesterday on win10, works *great. Is there any way this could work as a lock screen and not only as a screen saver?
@alistairmcmillan "Right click Aerial.scr and choose Install, windows will install it for you."

*a little buggy with multiple screens, but works

**Comment by danielscnc** [comment]
If only that option was there

**Comment by zarurc** [comment]
Did you unzip? I didn't have the option only until I unzipped.
If you did then sorry. maybe a screenshot?

**Comment by krzystoff** [comment]
This shows the only options I have for SCR files on Windows 10:
![Uploading aerialscreensaver windows10 options.JPG…]()

**Comment by alistairmcmillan** [comment]
@krzystoff Sorry could you try that again. It looks like the image didn't finish uploading.

**Comment by danielscnc** [comment]
unzip then right click the install option does not come up in the right click menu, it runs but does not install

**Comment by dglesne** [comment]
Well, I have the same issue.  I have no option to install.

**Comment by krzystoff** [comment]
I figured out how to get it working for me.  
I checked that I had .NET 4.6 and Windows Media Player installed (both installed by default with Windows 10).
The right-click install option was unavailable (AFAIK, scr install was disabled in Windows 10 for new installations, and only remains on upgraded OSes):  solution is to find the install location in your Windows\subfolders (search for *.scr from fil manager), in my case it was "C:\Windows\SysWOW64"
The screen saver dialog isn't easy to find, just press the Windows key and type 'screen saver' to open it.
Lastly the videos wouldn't download, the screen saver was selected but would not play any videos; we had some apple domains were blocked by my firewall, so I just removed them from the list, and the screensaver started perfectly!
I will batch download all the MOV files for all of our computers, to reduce bandwidth demands.
